src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,55 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String CHARACTER_S = "s";
+    private static final String CHARACTER_B = "b";
+    private static final String SPLIT_REGEX_COMMA = ",";
+    private static final int WORD_INDEX = 0;
+    private static final int AMOUNT_INDEX = 1;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String report = createReport(fromFileName);
```

Comment:
divide createReport() into to methods, lets read data from file in a separate method